navigation/Home/HomeTabNavigator.tsx

```diff
@@ -0,0 +1,123 @@
+/* eslint-disable react-native/no-color-literals */
+import * as React from 'react';
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
+import { View, Image, TouchableOpacity, StyleSheet } from 'react-native';
+import {OnboardingScreen} from '../../screens/Onboarding/OnboardingScreen';
+import {MyProfileScreen} from '../../screens/Profile/MyProfileScreen/MyProfileScreen';
+import {AuthenticationScreen} from '../../screens/Registration/AuthenticationScreen/AuthenticationScreen';
+//lib for react-native-svg
+import { Svg, Path } from 'react-native-svg';
+import { SvgProps } from 'react-native-svg';
+
+// Import your tab icons
+import { white, peach, tabBarFill } from '../../assets/colors/colors';
+
+import HomeIconFocused from '../../assets/icon-home-focused.svg';
+//import HomeIconUnfocused from '../../assets/icon-home-unfocused.svg';
+import ExploreIconFocused from '../../assets/icon-explore-focused.svg';
+import ExploreIconUnfocused from '../../assets/icon-explore-unfocused.svg';
+import AddIcon from '../../assets/add-icon.svg';
+
+const Tab = createBottomTabNavigator();
+
+
+
+
+
+const HomeTabNavigator = () => {
+  return (
+    <Tab.Navigator
+      screenOptions={({ route }) => ({
+        tabBarIcon: ({ focused }) => {
+          // Choose the icon based on the route name and focus state
+          let IconComponent : React.FC<SvgProps>;
+
+          if (route.name === 'Home') {
+            IconComponent = focused ? HomeIconFocused : HomeIconFocused;
+          } else if (route.name === 'Profile') {
+            IconComponent = focused ? ExploreIconFocused : ExploreIconUnfocused;
+          } else if (route.name === 'Add') {
+            IconComponent = AddIcon; // Assuming the same icon for focused and unfocused states
+          }
+
+          // Set the fill color based on focus state
+          //const fillColor = focused ? peach : tabBarFill;
+
+          // Return the icon component
+          return <IconComponent width={68} height={68} />;
+        },
+        tabBarActiveTintColor: peach,
+        tabBarInactiveTintColor: 'gray',
+        tabBarShowLabel: false,
+        tabBarStyle: styles.tabBar,
+      })}
+      initialRouteName="Home"
+    >
+      <Tab.Screen name="Home" component={OnboardingScreen} />
+      <Tab.Screen name="Add" component={AuthenticationScreen} />
+      <Tab.Screen name="Profile" component={MyProfileScreen} />
+    </Tab.Navigator>
+  );
+};
+
+
+
+const shadow = StyleSheet.create({
+  shadow: {
+    elevation: 5,
+    shadowColor: '#7F5DF0',
+    shadowOffset: {
+      width: 0,
+      height: 10,
+    },
+    shadowOpacity: 0.25,
+    shadowRadius: 3.5,
+    
+  },
+
+});
+const styles = StyleSheet.create({
+  
+  
+  addButton: {
+    // Custom styles for the 'Add' button
+    backgroundColor: white, // Example background color
+    borderRadius: 25,
+    marginVertical: -20,         // Example border radius to make it circular
```

Comment:
margin with a negative value is not valid

Reply:
corrected